Mathlib/Logic/Function/Basic.lean

```diff
@@ -434,6 +434,10 @@ theorem invFun_eq (h : ∃ a, f a = b) : f (invFun f b) = b :=
   by simp only [invFun, dif_pos h, h.choose_spec]
 #align function.inv_fun_eq Function.invFun_eq
 
+theorem apply_invFun_apply {α : Type u₁} {β : Type u₂} {f : α → β} {a : α} :
```

Comment:
Can this be simp?
```suggestion
@[simp] theorem apply_invFun_apply {α : Type u₁} {β : Type u₂} {f : α → β} {a : α} :
```

Reply:
Unfortunately it cannot, because the head symbol of the LHS is a variable.